modules/gdscript/gdscript.h

```diff
@@ -341,6 +341,8 @@ class GDScriptLanguage : public ScriptLanguage {
 	HashMap<StringName, int> globals;
 	HashMap<StringName, Variant> named_globals;
 
+	bool finalizing = false;
```

Comment:
Unused variable (I thought our warnings would catch this...).

Reply:
Removed it!